app/(afterLogin)/(tna)/tna/settings/approvals/_component/addApprover/index.tsx

```
@@ -66,6 +67,18 @@ const AddApprover = () => {
         onSuccess: () => {
           setAddModal(false);
         },
+        onError: (error: any) => {
+          // For AxiosError, the backend message is in error.response.data.message
+          const errorMessage = 
```

ESLint

[error] Delete `·`

(prettier/prettier)